python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -468,17 +468,18 @@ def _optimize_fp32_graph(self, graph):
 
     def _apply_pass(self, graph, pass_name, attrs=None, attr_values=None):
         ir_pass = core.get_pass(pass_name)
-        inference_program = graph.to_program()
-        ir_graph = core.Graph(inference_program.desc)
-        ir_graph.set_not_owned('__param_scope__', self._scope)
+        cpp_graph = graph.graph
+        if cpp_graph.has('__param_scope__'):
+            cpp_graph.erase('__param_scope__')
+        cpp_graph.set_not_owned('__param_scope__', self._scope)
         if attrs:
             assert attr_values and len(attrs) == len(
                 attr_values
             ), "Different number of pass attributes and their values."
             for attr, value in zip(attrs, attr_values):
                 ir_pass.set(attr, value)
-        ir_pass.apply(ir_graph)
-        graph = IrGraph(ir_graph, for_test=True)
+        ir_pass.apply(cpp_graph)
+        graph = IrGraph(cpp_graph, for_test=True)
```

Comment:
`graph = IrGraph(cpp_graph, for_test=True)` can be removed. Have a try please.

Reply:
If you remove this line, which means that after applying the pass, you changed Ir_graph to graph.graph, that doesn't make sense. Besides it will get the error 
``` ops = graph.all_op_nodes()
AttributeError: 'paddle.fluid.core_avx.Graph' object has no attribute 'all_op_nodes'
```